bigbluebutton-html5/imports/ui/components/whiteboard/shapes/helpers.js

```diff
@@ -0,0 +1,51 @@
+// applies zooming to the stroke thickness
+const zoomStroke = (thickness, widthRatio, heightRatio) => {
+  let ratio;
+
+  ratio = (widthRatio  + heightRatio) / 2;
+  return thickness * 100 / ratio;
+};
+
+const formatColor = (color) => {
+  //let color = this.props.shape.shape.shape.color;
+  if (color == null) {
```

Comment:
Might want to double-check that; my recollection (from the recording side) is that shape colors are actually sometimes interpreted as an integer value, and black is "0", which would be false in this check.